dagql/call/id.go

```diff
@@ -56,6 +56,13 @@ func (id *ID) Base() *ID {
 	return id.base
 }
 
+// The root Call of the ID, with its Digest set. Exposed so that Calls can be
+// streamed over the wire one-by-one, rather than emitting full DAGs, which
+// would involve a ton of duplication.
+func (id *ID) Call() *callpbv1.Call {
```

Comment:
Just FYI the reason I didn't add this originally is that it allows mutation of the underlying pb which would break the digest. But I'm fine with it in place as long as the callers pinky promise to not mutate it 😄 (may be worth a comment on the function)